src/main/resources/i18n/messages.properties

```diff
@@ -2202,12 +2207,13 @@ CartEmpty.heading=Whoops..Nothing in here!
 # ========================================================================================== #
 CartTools.menu.pipelines=Pipelines
 CartTools.menu.galaxy=Export to Galaxy
+CartTools.menu.samples=Sample Utilities
```

Comment:
Looks like this one can be removed as well

Reply:
Removed in a814cb808e5850ff37f6a7a75b15117fc0239d6b